js/sign/README.md

```diff
@@ -96,8 +104,11 @@ rm ed25519key.pem
 
 ## Release Notes
 
-### v0.0.2
+### v0.1.0
 
+- BREAKING CHANGE: Introducing the support for using different types of signing
```

Comment:
Is it really a breaking change if using `{ key: privateKey }` is still possible?

Reply:
Ah, I guess the fact that `IntegrityBlockSigner::sign` now returns a Promise is a breaking change.

Reply:
Ah, it looks like "Option 1" above is just outdated?

Reply:
You are right, looks like i forgot to add the awaits for both of them :P Fixed now.